app/styles/_core.less

```diff
@@ -571,6 +571,46 @@ label.checkbox {
   }
 }
 
+.compute-resource {
+  .resource-size {
+    display: flex;
+    height: 27px;
+    .resource-amount {
+      width: 80%;
+      input {
+        width: 100%;
+      }
+    }
+    .resource-unit {
+      width: 20%;
+      .ui-select-container {
+        height: 27px;
+      }
+      .ui-select-choices,
+      .ui-select-toggle {
+        width: 100%;
+      }
+    }
+  }
+}
+
+@media (max-width: @screen-sm-max) {
+  .compute-resource {
+    .inline-select {
+      // Spacing below input at mobile sizes.
+      margin-top: 5px;
+    }
+    .resource-size {
+      .resource-amount {
+        width: 70%;
+      }
+      .resource-unit {
+        width: 30%;
```

Comment:
Better to make these widths the default and then change them for larger sizes ("mobile first" styles). Maybe

```css
.compute-resource {
  .resource-size {
    display: flex;
    height: 27px;
    .resource-amount {
      width: 70%;
      input {
        width: 100%;
      }
    }
    .resource-unit {
      width: 30%;
      .ui-select-container {
        height: 27px;
      }
      .ui-select-choices,
      .ui-select-toggle {
        width: 100%;
      }
    }
    @media (min-width: @screen-sm-min) {
      .resource-amount {
        width: 80%;
      }
      .resource-unit {
        width: 20%;
      }
    }
    @media (min-width: @screen-md-min) {
      .resource-amount {
        width: 85%;
      }
      .resource-unit {
        width: 15%;
      }
    }
  }
}
```